### ISSUE TITLE
Some Android SDKs version 15.0.0 is not compatible with Android 12 (Api level 31+)

### ISSUE THREAD

**Issue by NinoDLC** [facebook/facebook-android-sdk#1114]
### Checklist before submitting a bug report

- [X] I've updated to the latest released version of the SDK
- [X] I've searched for existing [Github issues](https://github.com/facebook/facebook-android-sdk/issues)
- [X] I've looked for existing answers on [Stack Overflow](https://facebook.stackoverflow.com), the [Facebook Developer Community Forum](https://developers.facebook.com/community/) and the [Facebook Developers Group](https://www.facebook.com/groups/fbdevelopers)
- [X] I've read the [Code of Conduct](CODE_OF_CONDUCT.md)
- [X] This issue is not security related and can safely be disclosed publicly on GitHub

### Java version

11

### Android version

API 32

### Android SDK version

15.0.0

### Installation platform & version

Gradle

### Package

Gaming Services

### Goals

Upgrade from v14.1.1 (working) to v15.0.0 (not working) for gradle dependencies `facebook-android-sdk` and `facebook-login`

### Expected results

It compiles.

### Actual results

It doesn't compile. 

Error during compilation is 
```
C:\Work\foo\bar\app\src\main\AndroidManifest.xml Error:
	android:exported needs to be explicitly specified for <activity>. Apps targeting Android 12 and higher are required to specify an explicit value for `android:exported` when the corresponding component has an intent filter defined. See https://developer.android.com/guide/topics/manifest/activity-element#exported for details.
C:\Work\foo\bar\app\src\main\AndroidManifest.xml Error:
	android:exported needs to be explicitly specified for <activity>. Apps targeting Android 12 and higher are required to specify an explicit value for `android:exported` when the corresponding component has an intent filter defined. See https://developer.android.com/guide/topics/manifest/activity-element#exported for details.
```

### Steps to reproduce

Try to compile an Android project with : 
* `compileSdk` & `targetSdk` set to 31 (or more)
* `implementation com.facebook.android:facebook-android-sdk:15.0.0`

### Code samples & details

If I run `./gradlew :app:dependencies`, I can see this : 

```
+--- com.facebook.android:facebook-android-sdk:15.0.0
...
|    +--- com.facebook.android:facebook-gamingservices:15.0.0
...
|    |    +--- androidx.test:core:1.0.0
|    |    |    +--- androidx.annotation:annotation:1.0.0 -> 1.3.0
|    |    |    +--- androidx.test:monitor:1.1.0 -> 1.3.0-rc03 (*)
|    |    |    \--- androidx.lifecycle:lifecycle-common:2.0.0 -> 2.4.0 (*)
``` 

`androidx.test:core:1.0.0` is not compatible with Android 12 because it doesn't provide the necessary `android:exported` attribute for its Activities. 

The fix is to upgrade the androidx.test:core dependency in the facebook-gamingservices lib

**Comment by THMkjorl** [comment]
experienced this issue this morning also, decided to just downgrade to 14.1.1

**Comment by mingcaozhang** [comment]
Thanks for reporting this bug! We're working on issuing a hotfix right now.

**Comment by helloword1** [comment]
### add this to your manifest
`        <activity
            android:name="androidx.test.core.app.InstrumentationActivityInvoker$EmptyActivity"
            android:exported="false"
            />
        <activity
            android:name="androidx.test.core.app.InstrumentationActivityInvoker$EmptyFloatingActivity"
            android:exported="false"/>`

**Comment by Akilramki** [comment]
Downgrade the Android SDKs version less than 15.0.0

implementation 'com.facebook.android:facebook-android-sdk:14.1.1'

**Comment by Yusk03** [comment]
Also faced with this problem

**Comment by pastean** [comment]
When will this fix get released?

**Comment by mingcaozhang** [comment]
This issue is fixed in [release v15.0.1](https://github.com/facebook/facebook-android-sdk/releases/tag/sdk-version-15.0.1).
